client/src/components/editor/TextBox.js

```diff
@@ -6,7 +6,7 @@ class TextBox extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      text: ''
+      text: this.props.value
```

Comment:
Why mixing props and state? Generally an anti-pattern

Reply:
Also, will this break the create form?

Reply:
Yeah I shouldn't be doing it this way. Fixing this rn

Reply:
fixed this